.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -0,0 +1,120 @@
+      id-token: 'write'
+    outputs:
+      custom-version-json: ${{ steps.render-rawls-version.outputs.custom-version-json }}
+    steps:
+      - uses: 'actions/checkout@v3'
+
+      - name: Bump the tag to a new version
+        uses: databiosphere/github-actions/actions/bumper@bumper-0.2.0
+        id: tag
+        env:
+          DEFAULT_BUMP: patch
+          GITHUB_TOKEN: ${{ env.TOKEN }}
+          RELEASE_BRANCHES: main
+          WITH_V: true
+
+      - name: dispatch build to terra-github-workflows
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: rawls-build
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/main
+          token: ${{ env.TOKEN }}
+          inputs: '{ "repository": "${{ github.event.repository.full_name }}", "ref": "refs/heads/${{ inputs.branch }}", "rawls-release-tag": "${{ steps.tag.outputs.tag }}" }'
+
+      - name: Render Rawls version
+        id: render-rawls-version
+        env:
+          GITHUB_CONTEXT: ${{ toJSON(github) }}
+        run: |
+          echo "$GITHUB_CONTEXT"
+          echo "custom-version-json={\\\"rawls\\\":{\\\"appVersion\\\":\\\"${{ steps.tag.outputs.tag }}\\\"}}" >> $GITHUB_OUTPUT
+
+  create-bee-workflow:
+    runs-on: ubuntu-latest
+    needs: [rawls-build-tag-publish-job]
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: Echo Rawls version
+        run: |
+          echo '${{ needs.rawls-build-tag-publish-job.outputs.custom-version-json }}'
+
+      - name: dispatch to terra-github-workflows
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: bee-create
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/main
+          token: ${{ env.TOKEN }}
+          inputs: '{ "bee-name": "${{ env.BEE_NAME }}", "version-template": "dev", "custom-version-json": "${{ needs.rawls-build-tag-publish-job.outputs.custom-version-json }}" }'
+
+  rawls-swat-e2e-test-job:
+    runs-on: ubuntu-latest
+    needs: [create-bee-workflow]
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: dispatch to terra-github-workflows
+        env:
+          rawls_test_command: "testOnly -- -l ProdTest -l NotebooksCanaryTest -n org.broadinstitute.dsde.test.api.WorkspacesAzureTest"
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: .github/workflows/rawls-swat-tests.yaml
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/main
+          token: ${{ env.TOKEN }}
+          inputs: '{ "bee-name": "${{ env.BEE_NAME }}", "ENV": "qa", "ref": "refs/heads/${{ inputs.branch }}", "test-group-name": "workspaces_azure", "test-command": "${{ env.rawls_test_command }}", "java-version": "17" }'
+
+  destroy-bee-workflow:
+    runs-on: ubuntu-latest
+    needs: [rawls-swat-e2e-test-job]
+    if: always() # always run to confirm bee is destroyed
```

Comment:
Potential future improvement would be to parameterize this. It can be helpful to keep the bee around after failed test runs so we can poke around and see what went wrong. Out of scope for this PR though

Reply:
I agree this could be useful. If we find ourselves mucking this condition on a branch, let's definitely make it an input argument.